en/mkdocs.yml

```diff
@@ -736,6 +733,9 @@ nav:
       - 'Enabling HostName Verification': administer/enabling-hostname-verification.md
       - 'Configuring TLS Termination': administer/configuring-tls-termination.md
     - 'Getting WSO2 Updates': administer/getting-wso2-updates.md
+  - 'Compliance':
+    - 'General Data Protection Regulation': compliance/general-data-protection-regulation.md
+    - 'Electronic Identification, Authentication and Trust Services Regulation': compliance/electronic-identification-authentication-and-trust-services-regulation.md
```

Comment:
Let's add the abbreviations instead of the definitions for the left-navigation.

Reply:
modified